### PR TITLE
[FLOC-3876] Notify slack if a daily jenkins job fails.

### PR DESCRIPTION
**Pull request by james-w**
Jenkins is now configured to be able to notify slack. This adjusts
the job config of the daily jobs to send a notification to

### CODE REVIEW

**Comment by james-w** [comment]
You can see the notifications from my test instance on #engineering.

**Comment by james-w** [comment]
Oh, and this shouldn't land until I've deployed the changes to Jenkins, so
don't try and be helpful and click the button on this one please.

File: jobs.groovy [hunk 1]
```diff
@@ -427,6 +438,33 @@ def build_steps(job) {
...
+            notifyNotBuilt(true)
+            notifyUnstable(true)
+            notifyFailure(true)
+            notifyBackToNormal(false)
+            notifyRepeatedFailure(true)
+            includeTestSummary(true)
+            showCommitList(true)
+            includeCustomMessage(false)
+        }
+        it / 'publishers' << 'jenkins.plugins.slack.SlackNotifier'(plugin: 'slack@1.8.1')
```
Comment (jml) [inline jobs.groovy]: "We couldn't possibly use Haskell, no one would understand it." :(

(no change required)

**Comment by jml** [comment]
LGTM.

**Pull request merged**